modules/ossm-federation-create-import.adoc

```diff
@@ -95,7 +95,7 @@ For example:
 $ oc create -n green-mesh-system -f import-from-red-mesh.yaml
 ----
 +
-. Create additional `ImportServiceSets` as needed for each mesh peer in your federated mesh.
+. Create additional `ImportedServiceSets` as needed for each mesh peer in your federated mesh.
```

Comment:
Wonder if, since the resource is called ImportedServiceSet, we could revise this to say: 

. Create additional `ImportedServiceSet` resources as needed for each mesh peer in your federated mesh.

I don't have much of a preference--it was just a thought.